.github/workflows/eval-lib-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: "Evaluate Nixpkgs lib-tests"
+
+permissions:
+  contents: read
+
+on:
+  pull_request_target:
```

Comment:
Maybe we only want this if `./lib` changes? It should be self-contained. Also probably maybe it shouldn't be build on staging because it depends on pkgs.

Reply:
Done. https://github.com/JohnRTitor/nixpkgs/actions/runs/12140901171/job/33851723647?pr=1